operator/v1/types_console.go

```diff
@@ -114,7 +114,7 @@ type ConsoleCustomization struct {
 	// SVG format preferred
 	// +optional
 	CustomLogoFile configv1.ConfigMapFileReference `json:"customLogoFile,omitempty"`
-	// developerCatalog allows to configure the shown developer catalog categories.
+	// developerCatalog allows to configure the shown developer catalog categories and it's types.
```

Comment:
nit: its types or just types, or?

```suggestion
	// developerCatalog allows to configure the shown developer catalog categories (filters) and types (providers).
```